media/js/freesound.js

```diff
@@ -126,3 +125,51 @@ function show_hide_bookmark_form(id)
     }
     $("#bookmark_form_"  + id.toString()).toggle();
 }
+
+// FOOTER BANNER
+function hideFooterBanner(){
+    $('#footerBanner').hide(200);
+}
+
+function showFooterBanner() {
+    $('#footerBanner').show(200);
+}
+
+// Modals
+function hideModal(){
+    $("#fsmodal").hide();
+}
+
+function openModal(){
+    $("#fsmodal").show();
+}
+
+function setContentModalHTML(title, contents){
+  $('#fsmodal').find('.modal-header').html('<span class="close-modal">&times;</span>' + title);
+  $('#fsmodal').find('.modal-body').html(contents);
+  $("#fsmodal").find('.close-modal').click(hideModal);
+}
+
+
+// After download banner
+
+function afterDownloadModal(show_modal_url, sound_name){
+    hideModal(); // Hide the modal just in case it was shown
+
+    // Keep track of number of downloads this day
+    var numberDownloads = $.cookie("numberDownloads");
+    if (numberDownloads === undefined) {numberDownloads = 0;}
```

Comment:
Fixed in https://github.com/MTG/freesound/pull/820/commits/a0c044a6ca758b9c75906e450bd74e0ef2b12fec